src/storage.ts

```diff
@@ -641,6 +641,11 @@ export class Storage extends Service {
         delete body[storageClass];
       }
     });
+    if (body.storageClass && metadata.storageClass) {
```

Comment:
This will throw if a user provides only one storageClass as well, such as this:

```js
it('should allow a user-specified storageClass', done => {
  storage.createBucket(
    BUCKET_NAME,
    {
      storageClass: 'nearline',
    },
    done
  );
});
```
```
  1) Storage
       createBucket
         should allow a user-specified storageClass:
     Error: Both `nearline` and `storageClass` were provided.
      at Storage.createBucket (build/src/storage.js:309:19)
          -> /Users/stephen/dev/nodejs-storage/src/storage.ts:645:13
      at Context.<anonymous> (build/test/index.js:439:21)
          -> /Users/stephen/dev/nodejs-storage/test/index.ts:592:15
      at processImmediate (internal/timers.js:456:21)
```

Reply:
Done, and added the test.
Thanks.
PTAL